sphinx/cmdline.py

```diff
@@ -114,7 +114,7 @@ def handle_exception(app, opts, exception, stderr=sys.stderr):
                   file=stderr)
 
 
-def main(argv):
+def main(argv=sys.argv[1:]):
```

Comment:
Thanks, I've never used mypy before. Alternatively, I could remove the default argument and go back to the way things were before. I was just trying to make everything the same.